ddp-workspace/projects/ddp-dsm-ui/src/app/STORE/actions/enums/participants-enums.ts

```diff
@@ -0,0 +1,11 @@
+export enum getParticipants {
+  REQUEST = '[STORE] getParticipants Request',
```

Comment:
In such files are just written action types (according to the suggested convention) that later are used to distinguish different types of actions from each other.